source/includes/documentation/_html5_elements.md

```diff
@@ -4,74 +4,42 @@
 #### 1.1.1 Label
 
 ```java 
+  //In the following test 'name' is used to define locator
    
-  //In the following test, 'name' and 'disabledName' labels are found by utilizing their element locators:
-   
-  // @FindBy(css = "#name")
-  @UI("#name") 
-  public static TextField name;
-	
-  // @FindBy(css = "#disabled-name")
-  @UI("#disabled-name") 
-  public static TextField disabledName;
+  // @FindBy(xpath = "//label[@for='#name']")
+  @UI("//label[@for='#name']") 
```

Comment:
`#name` here was as if you used xpath `//*[@id='name']` i.e. `#`=`@id`. so you should do this as `@for='name'`

Reply:
also please delete excess linespace

Reply:
also please recheck xpath. I have not found any visible element by that xpath on a page. I guess you meant `//label[@for='your-name']` or `#your-name`